db/migrations/20180919153237_goteo_sdg.php

```diff
@@ -119,7 +119,15 @@ public function postUp()
     foreach($footprints[$sql_lang] as $id => $line) {
       $foot = new Footprint(['name' => $line[0]]);
       $errors = [];
-      if(!$foot->save($errors)) {
+      $fields = ['name', 'icon', 'description'];
+
+      try {
+          $foot->dbInsertUpdate($fields);
+      }
+      catch(\PDOException $e) {
+          $errors[] = 'Error saving footprint: ' . $e->getMessage();
+      }
+      if(!empty($errors)) {
```

Comment:
@davidbeig What's this migration change? Is it really related with this PR?

Reply:
Since the new migration for this two modules (the one that creates title for footprint), this migration is broken, that's why we create those footprints only using the name, icon and description. I guess that this commit is also inside the other PRs...